tracer/test/Datadog.Trace.Tests/LambdaCommonTests.cs

```diff
@@ -60,10 +65,11 @@ public void TestCreatePlaceholderScopeSuccessWithoutContext()
         {
             var tracer = TracerHelper.Create();
             var scope = LambdaCommon.CreatePlaceholderScope(tracer, null, null);
+
             scope.Should().NotBeNull();
-            scope.Span.TraceId.ToString().Should().NotBeNull();
-            scope.Span.SpanId.ToString().Should().NotBeNull();
-            scope.Span.Context.TraceContext.SamplingPriority.Should().Be(1);
+            scope.Span.TraceId128.Should().BeGreaterThan((TraceId.Zero));
+            scope.Span.TraceId.Should().BeGreaterThan(0);
+            scope.Span.SpanId.Should().BeGreaterThan(0);
```

Comment:
I think you may have accidentally deleted a line:

```suggestion
            scope.Span.SpanId.Should().BeGreaterThan(0);
            scope.Span.Context.TraceContext.SamplingPriority.Should().Be(1);
```

Reply:
See my reply to the other comment.